pkg/cmd/server/api/types.go

```diff
@@ -36,6 +38,13 @@ type NodeConfig struct {
 
 	// AllowDisabledDocker if true, the Kubelet will ignore errors from Docker.  This means that a node can start on a machine that doesn't have docker started.
 	AllowDisabledDocker bool
+
+	// PodManifestPath specifies the path for the pod manifest file(s)
+	// The path could point to a single file or a directory that contains multiple manifest files
+	// This is used by the Kubelet to create pods on the node
+	PodManifestPath string
```

Comment:
Any file paths have to be added to `GetNodeFileReferences`. Also add validation to `ValidateNodeConfig`